src/Console/Application.php

```diff
@@ -40,14 +40,15 @@
  */
 final class Application extends BaseApplication
 {
+    public const APP_NAME = 'PHP CS Fixer';
```

Comment:
```suggestion
    public const NAME = 'PHP CS Fixer';
```

Because we have `VERSION` and `VERSION_CODENAME`, not `APP_VERSION` and `APP_VERSION_CODENAME`, and it's somehow redundant when you look at `Application::APP_NAME`.

Reply:
I'm OK with both, I will apply to declaration and usage